### ISSUE TITLE
Question about azimuth

### ISSUE THREAD

**Issue by XALEX-123** [Walter0807/MotionBERT#76]
https://github.com/Walter0807/MotionBERT/assets/84762994/f486a806-0f78-4a1f-a1ae-29ff6cda0b9e

https://github.com/Walter0807/MotionBERT/assets/84762994/63845511-a55e-4f00-9e1e-dc4754cbe352

Hello, thank you for this amazing work!
I have some question about azimuth, I use a indoor bike riding as input, and if the azimuth=70, it seems normal, however, once I change azimuth to 0 that I want to display it in front view, the body truns out become a little crooked. Is that normal? If not how can I adjust it?
Sorry about bothering you, Sincerely ask.

**Comment by BiomechatronicsRookie** [comment]
I experienced a similar situation and was thinking of opening an issue too!

**Comment by Walter0807** [comment]
Hi, I did not notice this before. Which model are you using? It seems like the global orientation is somewhat wrong.

**Comment by XALEX-123** [comment]
I use FT_MB_release_MB_ft_h36m

**Comment by Walter0807** [comment]
How about other models (FT_MB_lite_MB_ft_h36m_global_lite)? Also, you can try interactive visualization where you can change the view-angle arbitrarily.

**Comment by XALEX-123** [comment]
yeah I also tried global lite version but somehow it's still happened. 
What is interactive visualization ? Could  you clarify clearly for me?

**Comment by Walter0807** [comment]
You can visualize the 3D pose for a single frame using matplotlib and interact with it with the mouse in a GUI (maybe with the help of x11, mobaxterm, or jupyter notebook depending on your OS).

**Comment by XALEX-123** [comment]
OK I get it, thank for your answering. By the way, i'm not being rude, but that issue metioned above can't be solved right away right? Or is there some guess or hint about this issue, like where can I find the global orientation?

**Comment by Walter0807** [comment]
Sorry I haven't got time to check that yet, maybe you can try interactive visualization first to see what happened. Trying different video samples might also be helpful to understand the issue.

**Comment by sandstorm12** [comment]
I am having the same problem. However, instead of extracting the 3D joints using the `infer_wild.py` script, I extracted the SMPL joints from the `infer_wild_mesh.py` script. In my use case it seems correct.

You can extract the joint positions by adding this line of code at the end of `infer_wild_mesh.py` script:
```python
np.save(osp.join(opts.out_path, 'joints.npy'), reg3d_all)
```

**Comment by lingfengnanoda** [comment]
> I am having the same problem. However, instead of extracting the 3D joints using the `infer_wild.py` script, I extracted the SMPL joints from the `infer_wild_mesh.py` script. In my use case it seems correct.
> 
> You can extract the joint positions by adding this line of code at the end of `infer_wild_mesh.py` script:
> 
> np.save(osp.join(opts.out_path, 'joints.npy'), reg3d_all)

Hello, do you know the corresponding relationship between these joint positions? I try to load the smpl model with unity and load the saved npy file to drive the animation. It seems that there is some misalignment. Do you have any way to use these npy files to restore the animation? I am completing my graduation project. Thank you for your help.

**Comment by Walter0807** [comment]
Closing due to inactivity. Feel free to reopen if needed.
